### PR TITLE
new: doc: #23: update and clarify information in README

### PR DESCRIPTION
**Pull request by jordanw-bq**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,52 @@
...
+**NOTE:** RubiX Admin currently only supports RPM-based Linux distributions.
+
 ## Installation
-`rubix-admin` should be installed on the cluster (Presto/Spark/Hadoop2) master
+
+RubiX Admin should be installed on the Presto/Spark/Hadoop2 cluster master node.
+
+### From Source
 
     git clone https://github.com/qubole/rubix-admin.git
```
Comment (abhishekdas99) [inline README.md]:
```
git clone https://github.com/qubole/rubix-admin.git
cd rubix-admin
virtualenv venv
```

File: README.md [hunk 2]
```diff
@@ -1,29 +1,52 @@
...
+
+### From PIP
+
+    pip install rubix-admin
+
+## Configuration
+RubiX Admin expects a configuration file at `~/.radminrc`. This file can be
+generated by running the following command:
 
     rubix-admin -h
```
Comment (abhishekdas99) [inline README.md]: i think its` rubix_admin -h` ? @jordanw-bq can you double check

File: README.md (hunk 1 above)
Comment (jordanw-bq) [reply README.md]: Updated instructions

File: README.md (hunk 2 above)
Comment (jordanw-bq) [reply README.md]: You are correct. Updated all command references to `rubix_admin`

**Pull request merged**